WooCommerce/UITestsFoundation/Screens/MyStore/PeriodStatsTable.swift

```diff
@@ -11,12 +11,17 @@ public final class PeriodStatsTable: ScreenObject {
         $0.cells["period-data-thisWeek-tab"]
     }
 
+    private let monthsTabGetter: (XCUIApplication) -> XCUIElement = {
+        $0.cells["period-data-thisMonth-tab"]
+    }
+
     private let yearsTabGetter: (XCUIApplication) -> XCUIElement = {
         $0.cells["period-data-thisYear-tab"]
     }
 
     private var daysTab: XCUIElement { daysTabGetter(app) }
     private var weeksTab: XCUIElement { weeksTabGetter(app) }
+    private var monthsTab: XCUIElement { monthsTabGetter(app) }
     private var yearsTab: XCUIElement { yearsTabGetter(app) }
```

Comment:
nit: these variables sound a bit misleading, since both "This Month" and "This Week" tabs show stats for days. It's fine to leave them though if the changes are too big 😓 
```suggestion
    private var todayTab: XCUIElement { daysTabGetter(app) }
    private var thisWeekTab: XCUIElement { weeksTabGetter(app) }
    private var thisMonthTab: XCUIElement { monthsTabGetter(app) }
    private var thisYearTab: XCUIElement { yearsTabGetter(app) }
```

Reply:
Agreed! Updated in f6258115b180c19216a2f834a977d3d6b0bb7967.